src/main/scala/gitbucket/core/api/ApiRepository.scala

```diff
@@ -39,11 +41,17 @@ object ApiRepository {
       forks = forkedCount,
       `private` = repository.isPrivate,
       default_branch = repository.defaultBranch,
-      owner = owner
+      owner = owner,
+      has_issues = issueCount > 0
```

Comment:
This is not the same meaning as GitHub.

![image](https://user-images.githubusercontent.com/17608272/91863237-de370f80-eca9-11ea-9a4c-c20712408166.png)

Reply:
The number of issues is irrelevant here.

Reply:
@SIkebe Thanks for your review! 
Oops! I totally mistook meanings...  So it means GitBucket can have only `true` statement...?

Reply:
>So it means GitBucket can have only true statement...?

You can disable issue tracking here in GitBucket.

![image](https://user-images.githubusercontent.com/17608272/91865034-e8f2a400-ecab-11ea-844c-99781807fe90.png)

Reply:
In that case, `repository.issues_option` becomes `DISABLE`.

![image](https://user-images.githubusercontent.com/17608272/91865792-b72e0d00-ecac-11ea-8f2a-93e261e98ead.png)

Reply:
Thank you so much for your info! that's helps a lot! 
So this API returns false when repository.issues_option is "DISABLE", got it!

Reply:
Yeah, I think that's the desired behavior 😉

Reply:
TBH, I wondered why GiHub returns `"has_issues": true` even if the repository has no issues at first .

Reply:
@SIkebe I fixed definition of `has_issue` in https://github.com/gitbucket/gitbucket/pull/2542/commits/3f33f998ca94ae0bc520ed53623618c8edd4627d